src/lib/base-query.ts

```diff
@@ -209,15 +209,14 @@ export class BaseQuery extends Pagination {
       requestParams = { ...this._queryParams, query: { ...this._parameters } };
     }
 
-    const getRequestOptions: any = { params: requestParams };
+    const getRequestOptions: any = requestParams;
```

Comment:
@vkalta We have changed the third param of getData function. It now takes an object having params and headers both. Kindly revert this line.

Reply:
I have reverted the changes @nadeem-cs

Reply:
I have reverted the changes @nadeem-cs